Sources/SwiftDocC/SwiftDocCMarkdownOutput/MarkdownOutputManifest.swift

```diff
@@ -0,0 +1,126 @@
+        /// One or more protocols to which a type conforms.
+        case conformsTo
+        /// One or more types that conform to a protocol.
+        case conformingTypes
+        /// One or more types that are parents of the symbol.
+        case inheritsFrom
+        /// One or more types that are children of the symbol.
+        case inheritedBy
+    }
+    
+    /// A relationship between two documents in the manifest.
+    ///
+    /// Parent / child symbol relationships are not included here, because those relationships are implicit in the URI structure of the documents. See ``children(of:)``.
+    package struct Relationship: Codable, Hashable, Sendable, Comparable {
+        
+        package let sourceURI: String
+        package let relationshipType: RelationshipType
+        package let subtype: RelationshipSubType?
+        package let targetURI: String
+        
+        package init(sourceURI: String, relationshipType: MarkdownOutputManifest.RelationshipType, subtype: RelationshipSubType? = nil, targetURI: String) {
+            self.sourceURI = sourceURI
+            self.relationshipType = relationshipType
+            self.subtype = subtype
+            self.targetURI = targetURI
+        }
+        
+        package static func < (lhs: MarkdownOutputManifest.Relationship, rhs: MarkdownOutputManifest.Relationship) -> Bool {
+            if lhs.sourceURI < rhs.sourceURI {
+                return true
+            } else if lhs.sourceURI == rhs.sourceURI {
+                return lhs.targetURI < rhs.targetURI
+            } else {
+                return false
+            }
+        }
+    }
+    
+    package struct Document: Codable, Hashable, Sendable, Comparable {
+        
+        /// The URI of the document
+        package let uri: String
+        /// The type of the document
+        package let documentType: DocumentType
+        /// The title of the document
+        package let title: String
+                
+        package init(uri: String, documentType: MarkdownOutputManifest.DocumentType, title: String) {
+            self.uri = uri
+            self.documentType = documentType
+            self.title = title
+        }
+                
+        package static func < (lhs: MarkdownOutputManifest.Document, rhs: MarkdownOutputManifest.Document) -> Bool {
+            lhs.uri < rhs.uri
+        }
+    }
+    
+    /// All documents in the manifest that have a given document as a parent, e.g. Framework/Symbol/property is a child of Framework/Symbol
+    package func children(of parent: Document) -> Set<Document> {
+        let parentPrefix = parent.uri + "/"
+        let prefixEnd = parentPrefix.endIndex
+        return documents.filter { document in
+            guard document.uri.hasPrefix(parentPrefix) else {
+                return false
+            }
+            let components = document.uri[prefixEnd...].components(separatedBy: "/")
+            return components.count == 1
+        }
+    }
```

Comment:
This method is only ever used in `testSymbolManifestChildSymbols()`. If it's not functionality that anything needs or uses, can we remove it?